spec/lib/scripts/collection_items_list_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/scripts/collection_items_list"
+
+RSpec.describe 'lib/scripts/collection_items_list.rb' do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:file) {
+    make_generic_file(user, { id: 'bogus-file-id', title: ['Test File'] })
+  }
+  let(:parent_collection) {
+    make_collection(user, { title: 'Test Collection'})
+  }
+
+  before do
+    parent_collection.members << file
+    parent_collection.save!
+  end
+
+  describe 'CollectionItemsList' do
+    subject { CollectionItemsList.new(parent_collection.id) }
+
+    it '#initialize' do
+      expect(subject.collection).to be_truthy
+      expect(subject.collection_items_csv_file).to be_truthy
+    end
+
+    it 'creates the file with sanitized title' do
+      expect(
+        File.basename(subject.collection_items_csv_file.path)
+      ).to eq("Test_Collection_items_list.csv")
+    end
+
+    describe '#get_items_and_add_to_csv' do
+      context 'collection with one file' do
+        it 'adds items data to Test_Collection_items_list.csv' do
+          subject.get_items_and_add_to_csv
+          csv_result = File.readlines(
+            "#{Rails.root}/lib/scripts/results/Test_Collection_items_list.csv"
+          )
+          expected_data = "#{parent_collection.title},"\
+                          "#{file.title.first},"\
+                          "https://digitalhub.northwestern.edu/files/#{file.id}\n"
+          expect(csv_result[1]).to eq(expected_data)
+        end
+      end
+
+      context 'collection with one file and one child collection with one file' do
+        let(:child_file) {
+          make_generic_file(user, { id: 'bogus-child-file-id', title: ['Test Child File'] })
+        }
+        let(:child_collection) {
+          make_collection(user, {title: 'Test Child Collection'})
+        }
+
+        before do
+          child_collection.members << child_file
```

Comment:
I think you can one line these as something like `child_collection.update!(members: [child_file])`

Reply:
I believe I had trouble trying it this way in the past due to this being an ActiveFedora object, and not ActiveRecord. It would be better to have it in one line, so I will try again.

Reply:
Oh, I didn't realize it was not ActiveRecord. I'm not familiar with ActiveFedora so I will leave it to you to decide the best course of action here.

Reply:
Yea, this didn't work 😞